upload/src/addons/SV/UserActivity/_output/template_modifications/public/svUserActivity_thread_view_a.json

```diff
@@ -0,0 +1,9 @@
+{
+    "template": "thread_view",
+    "description": "Adds activity containers (list) - part 1",
+    "execution_order": 10,
+    "enabled": true,
+    "action": "preg_replace",
+    "find": "#^(<div class=\"block.*?)(<xf:pagenav.*?\\/>)$#sim",
```

Comment:
The XF1 version was above `thread_view_above_messages` not inside the navblock. The nav block css is funky and I'ld like to avoid moving this code block into there.